<launch>
	<arg name="UAV" default="" />
	<arg name="camera" default="0" />
	<arg name="VI" default="0" />

	<node pkg="april_tag" name="april_tag_$(arg UAV)" type="april_tag_node" output="screen">
		<!-- remaps -->
		<remap from="/camera/image_raw" to="/$(arg UAV)/cam$(arg camera)/image_raw"/>
		<remap from="/april_tag_debug/output_video" to="/$(arg UAV)/cam$(arg camera)/output_video"/>
		<remap from="/april_tags" to="/$(arg UAV)/cam$(arg camera)/april_tags"/>
		<!-- <remap from="/tf" to="/tf_AT_cam0" /> -->

		<param name="camera" value="0" />
		<!-- tag_size_cm: size of edge of the black frame in cms -->
		<param name="tag_size_cm" value="23.9"/>
		<param name="tag_id" value="3" />

		<rosparam file="/home/mohito/catkin_ws/src/april_tag/cfg/$(arg UAV)_dwn.yaml"/> 

		<remap from="tf_cam0" to="/$(arg UAV)/tf_cam$(arg camera)" />

	</node>

</launch>
